clusterloader2/pkg/prometheus/manifests/dashboards/master-dashboard.dashboard.py

```diff
@@ -148,43 +173,57 @@
 ]
 
 APISERVER_PANELS = [
-    d.simple_graph("goroutines", 'go_goroutines{job="master", endpoint="apiserver"}'),
     d.simple_graph(
-        "gc rate", 'rate(go_gc_duration_seconds_count{job="master", endpoint="apiserver"}[1m])'
+        "goroutines",
+        'go_goroutines{job="master", endpoint="apiserver"}',
+        legend="{{instance}}",
+    ),
+    d.simple_graph(
+        "gc rate",
+        'rate(go_gc_duration_seconds_count{job="master", endpoint="apiserver"}[1m])',
+        legend="{{instance}}",
     ),
     d.simple_graph(
         "alloc rate",
         'rate(go_memstats_alloc_bytes_total{job="master", endpoint="apiserver"}[1m])',
         yAxes=g.single_y_axis(format=g.BYTES_PER_SEC_FORMAT),
+        legend="{{instance}}",
     ),
     d.simple_graph(
         "Number of active watches",
-        "sum(apiserver_registered_watchers) by (group, kind, instance)",
+        "sum(apiserver_registered_watchers) by (instance, group, version, kind)",
+        legend="{{instance}}: {{version}}.{{group}}.{{kind}}",
     ),
     d.simple_graph(
         "Watch events rate",
-        "sum(irate(apiserver_watch_events_total[1m])) by (version, kind, instance)",
+        "sum(irate(apiserver_watch_events_total[1m])) by (instance, group, version, kind)",
+        legend="{{instance}}: {{version}}.{{group}}.{{kind}}",
     ),
     d.simple_graph(
         "(Experimental) Watch events traffic",
```

Comment:
You can remove (Experimental). It's merged already and personally I find this very useful metric.